src/docs/cookbook/navigation/passing-data.md

```diff
@@ -8,24 +8,26 @@ next:
   path: /docs/cookbook/networking/fetch-data
 ---
 
-Oftentimes, we not only want to navigate to a new screen, but also pass some
-data to the screen as well. For example, we often want to pass information about
-the item we tapped on.
-
-Remember: Screens are Just Widgets&trade;. In this example, we'll create a List
-of Todos. When a todo is tapped on, we'll navigate to a new screen (Widget) that
+Often, you not only want to navigate to a new screen,
+but also pass data to the screen as well.
+For example, you might want to pass information about
+the item that's been tapped.
+
+Remember: Screens are Just Widgets&trade;.
+In this example, create a List of Todos.
```

Comment:
List -> list
(to be consistent with the rest of the page)

Maybe also Todo -> todo unless it's in code font? Or consistently use Todo (not todo).